### ISSUE TITLE
Error doing row selections when a column holds lists

### ISSUE THREAD

**Issue by flexatone** [static-frame/static-frame#784]
```python
>>> f = sf.Frame.from_records((('a', [1, 2]), ('b', [3, 4])))
>>> f.loc[1]
Traceback (most recent call last):
static_frame.core.exception.ErrorInitSeries: Index has incorrect size (got 2, expected 3)
```

**Comment by flexatone** [comment]
Thanks, @philipmassouh , for finding this!

**Comment by flexatone** [comment]
This is corrected; the fix will be part of the 1.3.2 release.
